cumulusci/tasks/preflight/tests/test_settings.py

```diff
@@ -1,5 +1,5 @@
 from unittest.mock import MagicMock
```

Comment:
This file seems like it should be promoted to a complete test class. Is there a good reason why it isn't one?

Reply:
pytest can run tests whether or not they are in a test class, and our coding style doesn't state a preference. What benefit do you see from putting them in a class?

Reply:
Since there's no stylistic preference, then it doesn't matter. I've usually seen tests collected into classes, but either way is fine.